src/redux/reducers/users.js

```diff
@@ -22,37 +23,27 @@ export const usersReducer = (state = initialState, action) => {
         ...state,
         users: action.payload
       };
-    case FOLLOW:
+    case FOLLOW_USERS_USER:
```

Comment:
CAn we name it just FOLLOW_USER? =)

Reply:
USERS is the name of the reducer.
FOLLOW_USER are too common names and can be used for other reducers.
We need to name the action type more uniqueway.
I am use the schema "[action]_[reducer]_[subject]" here.